data/mods/gen8bdsp/formats-data.ts

```diff
@@ -266,7 +266,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	politoed: {
 		tier: "UU",
 		doublesTier: "DOU",
-		randomBattleMoves: ["scald", "icebeam", "rest", "sleeptalk"],
+		randomBattleMoves: ["scald", "icebeam", "rest", "encore", "hypnosis"],
```

Comment:
Why remove Sleep Talk?

Reply:
We want politoed to either generate chesto berry+rest, or leftovers with the other moves; quite similarly to its gen8 rands moveset. Sleep talk+rest with scald and either hypnosis or encore wouldn't work very well.

Reply:
OK